docs/user-guide/production-pods.md

```diff
@@ -279,7 +279,7 @@ spec:
 			memory: 64Mi
 ```
 
-The container will die due to OOM (out of memory) if it exceeds its specified limit, so specifying a value a little higher than expected generally improves reliability. By specifying request, pod is guaranteed to be able to use that much of resource when needed. See [Resource QoS](../proposals/resource-qos.md) for the difference between resource limits and requests.
+The container will die due to OOM (out of memory) if it exceeds its specified limit, so specifying a value a little higher than expected generally improves reliability. By specifying request, pod is guaranteed to be able to use that much of resource when needed. See  for the difference between resource limits and requests.
```

Comment:
Looks like the munger munged all the links improperly. Please file an issue about that.